Dockerfile

```diff
@@ -8,4 +8,4 @@ ADD Pipfile Pipfile.lock ./
 
 RUN pipenv install --deploy --system
 
-ENTRYPOINT ["ipython"]
+CMD ["ipython"]
```

Comment:
What is the difference?

Reply:
Just to make it easier to override when needed. If we use `ENTRYPOINT` the user will have to override with the `--entrypoint` flag but if we use the CMD the user will be able just to pass the executable path to the run command.